### ISSUE TITLE
can I personalize the css? change colors?

### ISSUE THREAD

**Issue by sebasrock** [flauc/angular2-notifications#81]
can I personalize the css? change colors?

**Comment by flauc** [comment]
Yes `ViewEncapsulation` is set to none so you can override any style from the library.

**Comment by Sandybhai** [comment]
@flauc I am not able to locate it in any file. Could you guide me to that? As an aside, could this be added as an option parameter? 
Thanks.

**Comment by bukialabi** [comment]
Look for the notification component in your node modules:
node_modules/angular2-notifications/src/simple-notifications/components/notification.component.ts

**Comment by Zvax** [comment]
Small bump, it would be great if we could inject a global style object / stylesheet path when adding the module to the app in the imports array.

Directly editing the component in node_modules is a bad idea, as the code in there is hopefully not commited, and redeployment will overwrite it, as well as updates to the library, reinstalls of the modules suite, etc. etc.

**Comment by bukialabi** [comment]
The file I mentioned above shows you what the css selectors are and the default styles so you know what to change. What I've ended up doing in the meantime is creating a css file with my own color scheme and linking it at the bottom of my index page.
